process/factory/shard/vmContainerFactory.go

```diff
@@ -292,21 +302,21 @@ func (vmf *vmContainerFactory) createInProcessArwenVM_v1_3() (vmcommon.VMExecuti
 		DynGasLockEnableEpoch:    vmf.deployEnableEpoch,
 		ArwenV3EnableEpoch:       vmf.arwenV3EnableEpoch,
 	}
-	return arwenHost1_3.NewArwenVM(vmf.blockChainHookImpl, hostParameters)
+	return arwenHost13.NewArwenVM(vmf.blockChainHookImpl, hostParameters)
 }
 
-func (vmf *vmContainerFactory) createOutOfProcessArwenVM_v1_2() (vmcommon.VMExecutionHandler, error) {
+func (vmf *vmContainerFactory) createOutOfProcessArwenVMV12() (vmcommon.VMExecutionHandler, error) {
 	outOfProcessConfig := vmf.config.OutOfProcessConfig
-	logsMarshalizer := ipcMarshaling1_2.ParseKind(outOfProcessConfig.LogsMarshalizer)
-	messagesMarshalizer := ipcMarshaling1_2.ParseKind(outOfProcessConfig.MessagesMarshalizer)
+	logsMarshalizer := ipcMarshaling12.ParseKind(outOfProcessConfig.LogsMarshalizer)
+	messagesMarshalizer := ipcMarshaling12.ParseKind(outOfProcessConfig.MessagesMarshalizer)
 	maxLoopTime := outOfProcessConfig.MaxLoopTime
 
 	logger.GetLogLevelPattern()
 
-	arwenVM, err := ipcNodePart1_2.NewArwenDriver(
+	arwenVM, err := ipcNodePart12.NewArwenDriver(
```

Comment:
Not for this PR: we might think of a way to create wrapper structs for all these implementation variants in separate files: the main reason behind this is that we might change interfaces in the future (that the old, unsupported variants won't implement). These wrapper structs will then became adapters ensuring the code works as expected.